.codegen/changelog.md.tmpl

```diff
@@ -2,18 +2,29 @@
 
 ## {{.Version}}
 
-{{range .Changes -}}
+CLI:
+{{- range .Changes}}
  * {{.}}.
-{{end}}{{- if .ApiChanges}}
+{{- end}}
+
+Bundles:
+ * **FILL THIS IN MANUALLY BY MOVING RELEVANT ITEMS FROM ABOVE LIST**
```

Comment:
Manual change won’t get as git commit message. Should be something like “{{if contains . “ CLI “}}”, but you’re also required to modify the deco CLI

Reply:
Changing this in $EDITOR is possible in the most recent iteration.